### PR TITLE
New version: notepad-- version 2.6.0

### PR DESCRIPTION
**Pull request by hjkl950217**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/111423&drop=dogfoodAlpha

### CODE REVIEW

File: manifests/n/ndd/Notepad--/2.6.0/ndd.Notepad--.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: ndd.Notepad--
+PackageVersion: 2.6.0
+Installers:
+  - InstallerUrl: https://gitee.com/cxasm/notepad--/releases/download/v2.6/Notepad--v2.6.0-plugin-Installer.exe
```
Comment (CoolPlayLin) [inline manifests/n/ndd/Notepad--/2.6.0/ndd.Notepad--.installer.yaml:7]:
```suggestion
- InstallerUrl: https://gitee.com/cxasm/notepad--/releases/download/v2.6/Notepad--v2.6.0-plugin-Installer.exe
  InstallerLocale: zh-CN
```

Comment (CoolPlayLin) [inline manifests/n/ndd/Notepad--/2.6.0/ndd.Notepad--.installer.yaml:7]:
```suggestion
  - InstallerUrl: https://gitee.com/cxasm/notepad--/releases/download/v2.6/Notepad--v2.6.0-plugin-Installer.exe
    InstallerLocale: zh-CN
```

File: manifests/n/ndd/Notepad--/2.6.0/ndd.Notepad--.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    Architecture: x64
+    InstallerType: inno
+    InstallerSha256: 040E776B16E00D5CB5BA3FC4F5BCFB1615F1233FB5113808D93EBD00D13DBEF6
+    Scope: machine
+    InstallerSwitches:
+      Custom: /ALLUSERS
+  - InstallerUrl: https://gitee.com/cxasm/notepad--/releases/download/v2.6/Notepad--v2.6.0-plugin-Installer.exe
```
Comment (CoolPlayLin) [inline manifests/n/ndd/Notepad--/2.6.0/ndd.Notepad--.installer.yaml:14]:
```suggestion
  - InstallerUrl: https://gitee.com/cxasm/notepad--/releases/download/v2.6/Notepad--v2.6.0-plugin-Installer.exe
    InstallerLocale: zh-CN
```

File: manifests/n/ndd/Notepad--/2.6.0/ndd.Notepad--.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    Architecture: x64
+    InstallerType: inno
+    InstallerSha256: 040E776B16E00D5CB5BA3FC4F5BCFB1615F1233FB5113808D93EBD00D13DBEF6
+    Scope: user
+    InstallerSwitches:
+      Custom: /CURRENTUSER
```
Comment (CoolPlayLin) [inline manifests/n/ndd/Notepad--/2.6.0/ndd.Notepad--.installer.yaml:20]:
```suggestion
      Custom: /CURRENTUSER
  - Architecture: x64
    InstallerLocale: en-US
    InstallerType: inno
    InstallerUrl: https://github.com/cxasm/notepad--/releases/download/notepad--v2.6/Notepad--v2.6.0-plugin-Installer.exe
    InstallerSha256: 040e776b16e00d5cb5ba3fc4f5bcfb1615f1233fb5113808d93ebd00d13dbef6
    Scope: user
    InstallerSwitches:
      Custom: /CURRENTUSER
  - Architecture: x64
    InstallerLocale: en-US
    InstallerType: inno
    InstallerUrl: https://github.com/cxasm/notepad--/releases/download/notepad--v2.6/Notepad--v2.6.0-plugin-Installer.exe
    InstallerSha256: 040e776b16e00d5cb5ba3fc4f5bcfb1615f1233fb5113808d93ebd00d13dbef6
    Scope: machine
    InstallerSwitches:
      Custom: /ALLUSERS
```

**Review by CoolPlayLin — CHANGES_REQUESTED** [review]
你好 @hjkl950217 
这样应该更好

**Comment by CoolPlayLin** [comment]
image

File: manifests/n/ndd/Notepad--/2.6.0/ndd.Notepad--.installer.yaml (hunk 3 above)
Comment (hjkl950217) [reply manifests/n/ndd/Notepad--/2.6.0/ndd.Notepad--.installer.yaml:20]: 不错的建议

**Comment by CoolPlayLin** [comment]
你好 @hjkl950217 
没必要把 Master 合并到这里
还有
image
这个 Suggested change 一般都有 Commit suggestion 这个选项，可以直接提交，不用在本地复制在 Push

**Comment by hjkl950217** [comment]
> 你好 @hjkl950217 没必要把 Master 合并到这里 还有 image 这个 Suggested change 一般都有 Commit suggestion 这个选项，可以直接提交，不用在本地复制在 Push
好的，怕有冲突就会习惯性同步下再提交，感谢提醒

**Comment by hjkl950217** [comment]
> 你好 @hjkl950217 没必要把 Master 合并到这里 还有 image 这个 Suggested change 一般都有 Commit suggestion 这个选项，可以直接提交，不用在本地复制在 Push 请及时把 solve 这些 Commit suggestion

好的，怕有冲突就会习惯性同步下再提交，感谢提醒

**Comment by CoolPlayLin** [comment]
用 image
把 Commit suggestion 标记为已解决

**Review by stephengillie — APPROVED** [review]

**Pull request merged**